diesel_tests/tests/connection.rs

```diff
@@ -0,0 +1,18 @@
+use schema::connection;
+use diesel::*;
+use diesel::expression::dsl::sql;
+
+#[test]
+#[cfg(feature = "postgres")]
+fn managing_updated_at_for_table() {
+    let connection = connection();
+    connection.execute("CREATE TABLE yestest (n INTEGER, updated_at TIMESTAMP);").unwrap();
+    connection.execute("SELECT diesel_manage_updated_at('yestest');").unwrap();
+
+    connection.execute("INSERT INTO yestest (n) VALUES (2), (1), (5);").unwrap();
+    connection.execute("UPDATE yestest SET n = n + 1 WHERE true;").unwrap();
```

Comment:
While I know we can't test the exact value, can you test that this is setting the timestamp to a later time than before?